packages/@tinacms/toolkit/src/index.ts

```diff
@@ -39,6 +39,7 @@ export type { ContentCreatorPlugin } from './packages/forms'
 export * from './packages/forms'
 export * from './packages/icons'
 export * from './packages/react-dismissible'
+export { Nav } from './packages/react-sidebar'
```

Comment:
This is the only component that needs to be exported to `TinaAdmin`.  The rest of the `Sidebar` (and its context) are not necessary.